gulpfile.js

```diff
@@ -1,9 +1,20 @@
+import autoprefixer from 'autoprefixer';
+import browser from 'browser-sync';
+import { deleteAsync } from 'del'; 'del';
```

Comment:
Кое что лишнее. Должно быть так:
```suggestion
import { deleteAsync } from 'del';
```

Reply:
Исправлено